src/RoboFile/ClientContainer.php

```diff
@@ -33,7 +33,11 @@
 	public function runContainerTests(
 		$opts = array(
 			'env' => 'desktop',
-			'debug' => false
+			'debug' => false,
+			'single' => false,
```

Comment:
So we should make a separate robo script for `runContainerTestPreparation` in order to call it from the container startup script?